csvReader.js

```diff
@@ -0,0 +1,16 @@
+const { createReadStream, createWriteStream } = require('node:fs');
+const { pipeline } = require('node:stream/promises');
+const csv = require('csvtojson');
+
+class CsvReader {
```

Comment:
I guess this class is not only `csv` reader, it converts `csv` to `json`, so iin my opinion the name e.g. `CsvToJsonConverter` would be better, also i would rename method `read` -> `convert`